.github/workflows/main.yml

```
@@ -69,6 +70,14 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: us-east-1
 
+      - name: Test Google Life Sciences Executor
+        if: env.GOOGLE_APPLICATION_CREDENTIALS
```

```suggestion
        if: env.GCP_AVAILABLE
```

The setup gcloud step above enables the env var GOOGLE_APPLICATION_CREDENTIALS for all following steps. That should be all.